src/runtime/logging.cc

```diff
@@ -212,6 +212,13 @@ namespace tvm {
 namespace runtime {
 namespace detail {
 
+const char* ::tvm::runtime::detail::LogMessage::level_strings_[] = {
+    "Debug: ",    // TVM_LOG_LEVEL_DEBUG
+    "",           // TVM_LOG_LEVEL_INFO
```

Comment:
Why blank for info?

Reply:
This preserves the existing behavior where `LOG(INFO)` doesn't have anything prepended to it: https://github.com/apache/tvm/blob/afad20d8d9740baa1d251b7e80e8e56a1c2b7a4d/include/tvm/runtime/logging.h#L571-L575